SRC/GAME.CPP

```diff
@@ -2687,7 +2688,11 @@ int main( int argc, char *argv[] )
     srand( (unsigned) time( NULL ) );
     do_all();
     i::clear_stack();
+#ifndef TK_MACOSX_BUNDLE
```

Comment:
Wouldn't it be better to fix this in `parse_cmdline` by disabling the option there and also removing it from cmdline help for bundle builds?

Reply:
You're absolutely right.